src/logic/path.h

```diff
@@ -111,7 +111,7 @@ struct CoordPath {
 		return path_;
 	}
 
-	int32_t get_index(const Coords& field) const;
+	int32_t get_index(const Coords& c) const;
```

Comment:
in cases like this it might be better to use the more "telling name from the header in the .cc file and correct all imnstances of usage for better readability